ocs_ci/deployment/install_ocp_on_rhel.py

```diff
@@ -0,0 +1,211 @@
+"""
+This module will install OCP on RHEL nodes
+"""
+import logging
+import os
+
+from ocs_ci.framework import config
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources.pod import upload
+from ocs_ci.utility.templating import Templating
+from ocs_ci.utility.utils import (
+    create_rhelpod, get_cluster_version,
+)
+from semantic_version import Version
+
+
+logger = logging.getLogger(__name__)
+
+
+class OCPINSTALLRHEL(object):
+    """
+    Class to install OCP on RHEL nodes
+    """
+    def __init__(self, rhel_worker_nodes):
+        """
+        Initializes the required variables, create rhelpod, upload all the
+        helper files to pod and install required packages.
+
+        Args:
+            rhel_worker_nodes (list): list of RHEL nodes
+                e.g:['rhel1-0.vavuthu.qe.rh-ocs.com',
+                     'rhel1-1.vavuthu.qe.rh-ocs.com',
+                     'rhel1-2.vavuthu.qe.rh-ocs.com']
+
+        """
+        self.rhel_worker_nodes = rhel_worker_nodes
+        self.ssh_key_pem = config.DEPLOYMENT['ssh_key_private']
+        self.pod_ssh_key_pem = os.path.join(
+            constants.POD_UPLOADPATH,
+            self.ssh_key_pem.split("/")[-1]
+        )
+        self.ops_mirror_pem = os.path.join(
+            f"{constants.DATA_DIR}",
+            constants.OCP_PEM
+        )
+        self.cluster_path = config.ENV_DATA['cluster_path']
+        self.kubeconfig = os.path.join(
+            config.ENV_DATA['cluster_path'],
+            config.RUN.get('kubeconfig_location')
+        )
+        self.pod_name = "rhelpod"
```

Comment:
Create some name specific to this deployment.

Reply:
As discussed , raised issue https://github.com/red-hat-storage/ocs-ci/issues/1094 and will address in next PR.